presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improve performance of sum and average aggregations on decimals.
+* Add column comment to metadata in JDBC based connector
+* Add hidden column ``$file_modified_time`` which is the time the file containing the row was last modified.
+* Add hidden column ``$file_size`` which is the size of the file containing the row.
+* Add LocationProvider support to Iceberg connector so files are now written to paths defined by Iceberg's configured LocationProvider instead of a hard-coded table root directory.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix query failures for queries with shape ``AGG(IF(condition, expr))`` where expr could return exceptions for rows not matching ``condition``. These failures occurred when configuration property ``optimizer.aggregation-if-to-filter-rewrite-enabled`` was enabled.
+* Fix marking coordinator active when required resource managers (``cluster.required-resource-managers-active``, default value: 1) and coordinators (``cluster.required-coordinators-active``, default value: 1) are available in a cluster.
```

Comment:
Given that resource manager as a feature has not been made available - does this add value. @tdcmeehan ?

Reply:
Oops, TY!

Reply:
We'll add a release that links to documentation note once it's GA.  For now, let's remove this.

Reply:
Great, removed!